tests/network/services.go

```diff
@@ -170,6 +171,99 @@ var _ = SIGDescribe("Services", func() {
 			})
 		})
 	})
+
+	Context("Masquerade interface binding", func() {
+		var inboundVMI *v1.VirtualMachineInstance
+		var serviceName string
+
+		const (
+			selectorLabelKey   = "expose"
+			selectorLabelValue = "me"
+			servicePort        = 1500
+		)
+
+		createReadyVMIWithMasqueradeBindingAndExposedService := func(hostname string, subdomain string) *v1.VirtualMachineInstance {
+			vmi := libvmi.NewCirros(
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()))
+			return readyVMI(
+				exposeExistingVMISpec(vmi, subdomain, hostname, selectorLabelKey, selectorLabelValue))
+		}
+
+		BeforeEach(func() {
+			subdomain := "vmi"
+			hostname := "inbound"
+
+			inboundVMI = createReadyVMIWithMasqueradeBindingAndExposedService(hostname, subdomain)
+
+			tests.StartTCPServer(inboundVMI, servicePort)
+		})
+
+		Context("with a service matching the vmi exposed", func() {
+			var isDualStack bool
+			var ipv6ServiceName string
+
+			BeforeEach(func() {
+				serviceName = "myservice"
+
+				service := buildServiceSpec(serviceName, servicePort, servicePort, selectorLabelKey, selectorLabelValue)
+				_, err := virtClient.CoreV1().Services(inboundVMI.Namespace).Create(service)
+				Expect(err).ToNot(HaveOccurred())
+
+				isDualStack = flags.IsDualStackedNetworkConfig
```

Comment:
Hasn't this one taken out of this PR?

Reply:
Good catch. Looks like I had left this behind here.

Done.